doc/api/cookies.rst

```diff
@@ -0,0 +1,71 @@
+
+.. _cookies:
+Cookies
+-------------
+
+Cookie support has been added to falcon from version 0.3.
+
+.. _getting-cookies:
+Getting Cookies
+~~~~~~~~~~~~~
+
+Cookie can be read from a request via the :py:attr:`~.Request.cookies` request attribute:
+
+
+.. code:: python
+
+    class Resource(object):
+        def on_get(self, req, resp):
+
+            cookies = req.cookies
+
+            if "my_cookie" in cookies:
+                my_cookie_value = cookies["my_cookie"]
+            # ....
+
+The :py:attr:`~.Request.cookies` attribute is a regular
+:py:class:`dict` object.
+
+.. tip :: :py:attr:`~.Request.cookies` returns a
+    copy of the response cookie dict. Assign it to a variable as in the above example
+    for better performance.
+
+.. _setting-cookies:
+Setting Cookies 
+~~~~~~~~~~~~~~~
+
```

Comment:
I think it would be helpful to reiterate here why it is necessary to set cookies using this method, as opposed to `set_header` and `append_header`. In the former case, it is because multiple instances of the same header cannot be set. In the laster case it is because it uses the list syntax which is incompatible with Set-Cookie (see also: http://tools.ietf.org/html/rfc7230#section-3.2.2)